res/values-sq/strings.xml

```diff
@@ -32,15 +32,15 @@
 
     <string name="Content_rating_of_the_app_may_change_when_finding_groups_">"përmbajtja vlerësim të app mund të ndryshojë kur gjetjen e grupeve"</string>
 
-    <string name="_Rank_">"gradë"</string>
-    <string name="_Alias_">" pseudonim "</string>
-    <string name="_Score_">" Pikë "</string>
-    <string name="Score__">"Score: "</string>
+    <string name="_Rank_">"_Gradë_"</string>
+    <string name="_Alias_">"_Pseudonim_"</string>
+    <string name="_Score_">"_Pikë_"</string>
```

Comment:
Lines from 35 to 37: Why inserted bottom line? Please remove them, you don't need to base off on these lines.